src/ts/Profile/UserOptional.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { Paper } from 'material-ui';
+import { FormsyText } from 'formsy-material-ui/lib';
+import i from '../i18n';
+
+export default function UserOptional(props) {
+  const { user } = props;
+  const disabled = !user;
+  return (
+    <Paper className="user-form">
+      <div className="head">
+        {i('Optional Information')}
```

Comment:
there is a map english-string to polish-string and module that checks browser language and chooses right translation (checkout `src/ts/i18n`)